backend/api/Database/Models/Robot.cs

```diff
@@ -7,31 +7,31 @@ namespace Api.Database.Models
     public class Robot
     {
         [Key]
-        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         [Required]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public string Id { get; set; }
 
-        [MaxLength(128)]
         [Required]
+        [MaxLength(200)]
         public string Name { get; set; }
 
-        [MaxLength(128)]
         [Required]
         public RobotModel Model { get; set; }
 
-        [MaxLength(128)]
         [Required]
+        [MaxLength(200)]
         public string SerialNumber { get; set; }
 
+        [MaxLength(200)]
         public string Logs { get; set; }
 
         public float BatteryLevel { get; set; }
 
         [Required]
```

Comment:
Should this be required?

Reply:
I think that a "Required" here only requires there to be a list that can be empty. This changes how VideoStreams are initialized I believe. But I agreed that VideoStreams should not be required